src/foam/nanos/theme/Theme.js

```diff
@@ -613,6 +613,13 @@ foam.CLASS({
       },
       includeInDigest: true
     },
+    {
+      class: 'Array',
```

Comment:
Does this default to value of non-null?  Assuming, as no check above, but making sure.

Reply:
@jlhughes it does default to null, we do a `theme.getFindRestrictedCapabilities() != null` above to check for this case.